lib/laa/fee_calculator/version.rb

```diff
@@ -2,8 +2,8 @@
 
 module LAA
   module FeeCalculator
-    VERSION = "1.1.0"
-    VERSION_RELEASED = "2019-12-20"
+    VERSION = "1.2.0"
+    VERSION_RELEASED = "2021-02-01"
```

Comment:
once in master its worth changing this date to the date we are intending to release as a separate quick PR